### DISCUSSION TITLE
Issue with the filter Videodelay (async)

### DISCUSSION THREAD

**Discussion by djmisky** [DistroAV/DistroAV#955] [General]
Hey,

i got a new PTZ Camera the Obsbot "Tail Air".
when i try to activate the filter "Videodelay (async)" the delay is 10 seconds! not milliseconds.

i verified the problem on different OBS (28-30) and Plugin Versions and on 3 differnt Windows PCs.
Its always the same. 

With other NDI Cameras there is no problem like this.

my current workaround is to run a second OBS and send the signal to the NDI Main output and get this in my "normal" Streaming OBS installation.

can you help me?

**Comment by paulpv** [comment]
I so want one of those Tail Airs!
I even emailed them and said I am a dev for the obs-ndi project with millions of users (per https://obsproject.com/forum/resources/obs-ndi-newtek-ndi%E2%84%A2-integration-into-obs-studio.528/ we have 2,836,301 downloads) and that we would like to test with their NDI|HX camera.
We really are getting reports of NDI|HX issues, so I see this as a legit request.

Maybe I will ask them again.

Or maybe you can send me your camera! ;)

**Comment by paulpv** [comment]
w00t!
They replied to my follow up and they are sending us/me an Obsbot "Tail Air".
With this I can do some actual NDI|HX testing!

**Reply by djmisky** [reply]
perfect, then we will see if the problem is only with my camera or if its a general problem of this cam
